src/v2/Components/Address/AddressModalFields.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react"
+import { Input, Spacer, Text } from "@artsy/palette"
+import { FormikProps } from "formik"
+import { SavedAddressType } from "v2/Apps/Order/Utils/shippingAddressUtils"
+import { CountrySelect } from "../CountrySelect"
+
+interface Props {
+  formik: FormikProps<SavedAddressType>
+}
+
+export const AddressModalFields: React.FC<Props> = props => {
+  const { formik } = props
```

Comment:
Rather than passing, I believe you can use the `useFormikContext` hook to retrieve field stuff

Reply:
I was pleasantly surprised by how easier it is to use `useFormikContext` 🙏🏽

Reply:
yayy 👍